### ISSUE TITLE
There's a new Bugfender SDK version

### ISSUE THREAD

**Issue by andrei-tofan** [bugfender/rn-bugfender#74]
I've noticed this error in the build log although we are on the latest version: 2.1.2

```
There's a new Bugfender SDK version. Please check bugfender.com.
```
Does the iOS library need to be updated?

**Comment by jgimenez** [comment]
Yes, you can update the Podfile to the latest version if you'd like. We issue updates for the RN wrapper regularly, so eventually we'll update it here. If you'd like to submit a PR, you're welcome to do so!

**Comment by jgimenez** [comment]
I just checked that the dependency is indicated as `~> 1.10`, which means that the latest pod version as we speak (`1.12.1`) is included. I verified that by checking the lockfile.

If your project uses an older version of the pod, you may want to manually issue a `pod install` to get the latest version.
